presto-main/src/test/java/io/prestosql/server/TestDynamicFilterService.java

```diff
@@ -296,12 +346,33 @@ public void testReplicatedDynamicFilter()
                 singleValue(INTEGER, 1L))));
         assertTrue(dynamicFilter.isComplete());
         assertEquals(dynamicFiltersStageSupplier.getRequestCount(), 1);
+        DynamicFiltersStats stats = dynamicFilterService.getDynamicFilteringStats(queryId, session);
+        assertEquals(stats.getNumTotalDynamicFilters(), 1);
+        assertEquals(stats.getNumDynamicFiltersCompleted(), 1);
+        assertEquals(stats.getNumReplicatedDynamicFilters(), 1);
+        assertEquals(stats.getNumRepartitionedDynamicFilters(), 0);
+        assertEquals(
+                stats.getDynamicFilterDomainStats(),
+                ImmutableList.of(new DynamicFilterDomainStats(filterId1, Domain.create(
+                        SortedRangeSet.copyOf(INTEGER, ImmutableList.of(equal(INTEGER, 1L))),
+                        false)
+                        .toString(session.toConnectorSession()), 1, 0)));
 
         // all dynamic filters have been collected, no need for more requests
         dynamicFilterService.collectDynamicFilters();
         assertEquals(dynamicFiltersStageSupplier.getRequestCount(), 1);
     }
 
+    private static String getExpectedDomainString(Type type, Object low, Object high)
```

Comment:
type is always `INTEGER`, no need to pass it as arg. You can actually change signature to:
```
String getExpectedDomainString(int low, int high)
```